### ISSUE TITLE
Accessibility: The highlighted links below does not indicate whether they are internal or external links and whether they open in new window or not

### ISSUE THREAD

**Issue by ericmaino** [microsoft/BotFramework-WebChat#3087]
![image](https://user-images.githubusercontent.com/1005609/78628707-cd44b800-7849-11ea-8dfa-33ac39e93913.png)

**Actual**
In LHS screenshot.
- The first link is internal link and opens in a new window. There is no title attribute = “opens in a new window”. Also “opens in a new window” icon is not present.
- The second and third links are external links and opens in a new window. There is no title attribute = “opens in a new window, external”.  
 
**Expected**
Add title attribute as mentioned above and also use icons to represent “external links” and “links opening in new window” 
 
**WCAG Violation**
Predictable
https://www.w3.org/TR/WCAG20-TECHS/G201.html

**Comment by corinagum** [comment]
These are accessibility issues of v3 of Web Chat, which has been deprecated and we no longer support unless there are security risks. For accessible Web Chat, please [migrate to v4](https://github.com/Microsoft/BotFramework-WebChat#migrating-from-web-chat-v3-to-v4).

**Comment by ericmaino** [comment]
This issue is still present in v4. I'll investigate and propose submit a PR with a fix

**Comment by compulim** [comment]
In Web Chat v4, all links will be opened externally in a new tab. Please let us know if this is not true.

**Comment by JorgeCupiV** [comment]
While they open externally, there's still a need for the title property to show that the link is being open externally.

**Comment by Quirinevwm** [comment]
UX recommendation:
For communication purposes definitely include the transcript _"opens link in a new window"_, Whenever there is a deep link include this transcript for accessibility, since both types (internal, external) will have this behavior.

Regarding the icon ask:
- Stick to deep link only, **since all links will be opened in a new window in V4, there is no difference between "internal" and "external" links regarding behavior**. Hence, why I would recommend to not place any icon there. The content will look messy,  when after every link (both internal, external) we position a hovering icon that represents opening. Also when multiple "categorized" links will be places after _, link, link,_.

- If you do really need to position an icon after every link, go for the The Fluent System library icon represents "open":
![image](https://user-images.githubusercontent.com/43533472/105725736-9a2efa80-5f29-11eb-89fd-e76db1f17790.png)
https://iconcloud.design/search/filter/Fluent%20System%20Library/Fluent%20Regular/open/8406032d3-3821b375b

**Comment by compulim** [comment]
Modify `renderMarkdown.js`, some code in this snippets:

![image](https://user-images.githubusercontent.com/1622400/110706134-3b0d0880-81ac-11eb-892f-a58ae53f5d62.png)

It is easy to add `aria-label` (or other ARIA attributes), unsure if it is easy to add an icon in the Markdown tree.

**Comment by corinagum** [comment]
note to self: timebox to see how to add HTML elements for aria-label and icon
